README.md

```diff
@@ -1,15 +1,62 @@
-[![TLA+ Build Status](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
+Overview
+--------
+[![](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
 
-For more information, visit http://lamport.azurewebsites.net/tla/tla.html
+This repository hosts the core TLA⁺ command line interface (CLI) tools and the Toolbox integrated development environment (IDE).
+Its development is managed by the [TLA⁺ Foundation](https://foundation.tlapl.us/).
+See http://tlapl.us for more information about TLA⁺ itself.
+For the TLA⁺ proof manager, see https://github.com/tlaplus/tlapm.
+
+Versioned releases can be found on the [Releases](https://github.com/tlaplus/tlaplus/releases) page.
+Currently, every commit to the master branch is built & uploaded to the [1.8.0 Clarke pre-release](https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0).
+If you want the latest fixes & features you can use that pre-release.
+
+The CLI tools and Toolbox IDE are both written in Java.
+The Toolbox is based on the [Eclipse Platform](https://github.com/eclipse-platform/eclipse.platform).
+The CLI tool source code is primarily located in the [tlatools/org.lamport.tlatools](./tlatools/org.lamport.tlatools) directory.
+The Toolbox IDE source code is contained within the [toolbox](./toolbox) directory.
+
+Build & Test CLI Tools
```

Comment:
This section and the next one ("Build & Test Toolbox IDE") duplicate information in DEVELOPING.md. I would prefer to combine these sections into one named something like "Compile from Source" that just links to DEVELOPING.md.

Reply:
I personally like the style of having basic build/test instructions in project READMEs and was planning on turning DEVELOPING.md into more of a guide on how to set up a development environment with IDEs and such

Reply:
I think that suggestion makes sense. However, you might consider embedding specific code/commands from `DEVELOPING.md` into this section using [permalinks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet#linking-to-markdown).

Please note that any markdown will not be rendered in the included snippet.

Reply:
@ahelwer That makes sense to me. In that case, maybe we could make this section maximally copy-pasteable for readers who just want a quickstart:
```bash
git clone https://github.com/tlaplus.git
cd tlaplus/tlatools/org.lamport.tlatools
ant -f customBuild.xml compile dist # generates dist/tla2tools.jar
ant -f customBuild.xml compile-test test # run tests (optional)
java -jar dist/tla2tools.jar ./test-model/pcal/Bakery.tla # run an example
```
(I'm imagining that would replace everything from "Clone this repo" to `Bakery.tla`, and would be followed by a link to DEVELOPING.md for more info. I don't mind having many links to that file in the readme.)

(Also, we should double-check these instructions... it is (or used to be) true that you need to run `info` and `compile-test` in addition to or before the `dist` target.)

@juliaferraioli Interesting! I think it might be too brittle (we'd have to be updating line numbers in those links a lot), but it's an option to keep in mind.

Reply:
That seems good although I think I'll omit the git clone, because people might want to clone it in some other way (using the SSH address for example).

Reply:
I also double-checked on a fresh clone and yes `compile-test` is necessary to run `dist`. `info` is not. This is kind of weird, why do we need to compile the tests to build the distributable?

Reply:
Because some of the test infrastructure gets included in the final jar; see commit f7ea7a6219f18db9f69d4a296abec597bf8264c6.

This is something I'd like to fix eventually, but it isn't urgent. The right fix might be somewhat complicated.